Mathlib/Algebra/Category/FiniteGrp/Basic.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Nailin Guan, Yuyang Zhao
+-/
+import Mathlib.Algebra.Category.Grp.Basic
+
+/-!
+
+## Main definitions and results
+
+* `FiniteGrp` is the type of finite groups.
+
+-/
+
+universe u v
+
+open CategoryTheory
+
+/--Defining the structure of finite group-/
+@[pp_with_univ]
+structure FiniteGrp where
```

Comment:
I'm not very familiar with the algebraic hierarchy in Mathlib but, why not just use `[Group G] [Finite G]`? Also, is it intentional that there's no actual requirement `[Group toGrp]`?

Reply:
> I'm not very familiar with the algebraic hierarchy in Mathlib but, why not just use `[Group G] [Finite G]`? Also, is it intentional that there's no actual requirement `[Group toGrp]`?

This defines the category of finite groups. Maybe the annotation here should be revised to be clearer, following the example of the annotation in `Grp`.

Reply:
I think it is the doc string problem, it should be changed to `The category of finite groups`

Reply:
> I'm not very familiar with the algebraic hierarchy in Mathlib but, why not just use `[Group G] [Finite G]`? Also, is it intentional that there's no actual requirement `[Group toGrp]`?

This is to make `InducedCategory.category FiniteGrp.toGrp` possible.